conman/routes/handlers.py

```diff
@@ -1,4 +1,7 @@
-from django.core.urlresolvers import resolve
+from conman.compat import ignore_django20_deprecation
+
+with ignore_django20_deprecation():
+    from django.core.urlresolvers import resolve
```

Comment:
Although, in particular because of the `#noqa`, I think I would be most comfortable with this in the `compat` module.